src/main/java/com/google/sps/data/Term.java

```diff
@@ -0,0 +1,64 @@
+package com.google.sps.data;
+
+import java.util.*;
+
+public class Term {
+  String season;
+  int year;
+  String school;
+  ListIterator termIterator;
+  int numTerms;
+  int currTerm;
+  ArrayList<String> termList;
+
+  final List<String> QTR_SCHOOLS = Arrays.asList("caltech", "calpoly", "stanford", "berkeley");
+  final ArrayList<String> QTR_TERMS =
+      new ArrayList<String>(Arrays.asList("Winter", "Spring", "Fall"));
```

Comment:
what about summer?

Reply:
Wanted to discuss this with you actually - just sent a message in the chat about whether we should include summer or not